src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) throws IOException {
+        Files.write(Paths.get(filePath), data.getBytes());
```

Comment:
The `write` method should be used with try-with-resources or at least should handle possible exceptions more gracefully than just throwing it further. According to the checklist, it's recommended to catch an IOException and rethrow it as a RuntimeException with an informative message.